kemdk/build.gradle

```diff
@@ -11,7 +11,7 @@ plugins {
     id 'maven-publish'
 }
 
-version = "0.0.1"
+version = "1.0.1"
```

Comment:
Releases are based on git tag. This might have no effect at all. Do we need to keep it for reasons?

Reply:
Since it is not used, we remove it from code to avoid having a version number that we need to bump for documentation purpose only